### ISSUE TITLE
Date is not parsed to ISO string in DateTimeInput 

### ISSUE THREAD

**Issue by salony75way** [DeepBlueCLtd/RCO#261]
In react-admin date time input's parse property was not working.

Testcase: 
I have passed `parse` method to `DateTimeInput` in item's `CoreForm` to parse date in ISO string but in db it was saved as a Date object.

DateTimeInput docs: https://marmelab.com/react-admin/DateTimeInput.html

**Comment by IanMayo** [comment]
@salony75way - is this still an issue?

**Comment by salony75way** [comment]
> @salony75way - is this still an issue?

Yes we still have this. Currently we have implemented `convertDateToISO` method in life cycle method to fix this. react-admin `parse` method still not working for date parsing from Date object to ISO string.

**Comment by TahaKhanAbdalli** [comment]
Date is being stored in `ISO format` while creating item. @IanMayo

**Comment by IanMayo** [comment]
So it's not an issue?

**Comment by TahaKhanAbdalli** [comment]
No
